crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/staticdata/enums/converter/CourtTypeConverter.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.laa.crime.meansassessment.staticdata.enums.converter;
+
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.CourtType;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+/**
+ * JPA to Enum converter class. It is used when persisting CourtType enum to DB and when loading CourtType enum from DB
+ */
+@Converter(autoApply = true)
+public class CourtTypeConverter implements AttributeConverter<CourtType, String> {
```

Comment:
Do we really need this class?

Reply:
Hello @atharmajeed 
The enum converter is used for mapping enum value to DB field.
That way, when an enum is referenced from an JPA entity object, it can be referenced as any other primitive value. The converter will take care of saving  a string value onto the DB, and mapping a DB string value to the enum, when retrieving data from the DB.
I added it there so that when we do need to reference them in an entity, the functionality will already be there. I’ve documented  the functionality on : https://dsdmoj.atlassian.net/wiki/spaces/ASLST/pages/3787358301/Migrating+static+Data+to+Enum+-+Crime+Means+Assessment


However, as CourtType is not immediately needed, I'm happy to remove it and the relevant attribute converter companion class